### PR TITLE
add callout detailing what Zipf's law

### PR DESCRIPTION
**Pull request by psteinb**
might be helpful for instructors like me that never heard about Zipf's law

### CODE REVIEW

File: _episodes/11-snakemake-intro.md [hunk 1]
```diff
@@ -154,6 +154,15 @@ isles	3822	2460	1.55
...
+>
+> Zipf's law was originally formulated in terms of [quantitative linguistics](https://en.wikipedia.org/wiki/Quantitative_linguistics), stating that given some [corpus](https://en.wikipedia.org/wiki/Text_corpus) of [natural language](https://en.wikipedia.org/wiki/Natural_language) utterances, the frequency of any word is [inversely proportional](https://en.wikipedia.org/wiki/Inversely_proportional) to its rank in the [frequency table](https://en.wikipedia.org/wiki/Frequency_table). ... For example, in the [Brown Corpus](https://en.wikipedia.org/wiki/Brown_Corpus) of American English text, the word the is the most frequently occurring word, and by itself accounts for nearly 7% of all word occurrences (69,971 out of slightly over 1 million). True to Zipf's Law, the second-place word of accounts for slightly over 3.5% of words (36,411 occurrences), followed by and (28,852). Only 135 vocabulary items are needed to account for half the [Brown Corpus](https://en.wikipedia.org/wiki/Brown_Corpus).
```
Comment (tkphd) [inline _episodes/11-snakemake-intro.md]: - Please wrap text at 100 chars.
- Stray ellipsis `...` before "For example..."

File: _episodes/11-snakemake-intro.md [hunk 2]
```diff
@@ -154,6 +154,15 @@ isles	3822	2460	1.55
...
+> 
+> Source [Wikipedia](https://en.wikipedia.org/wiki/Zipf%27s_law):
```
Comment (tkphd) [inline _episodes/11-snakemake-intro.md]: `Source Wikipedia:` --> `Source: Wikipedia`

File: _episodes/11-snakemake-intro.md [hunk 3]
```diff
@@ -154,6 +154,15 @@ isles	3822	2460	1.55
 ```
 {: .output}
 
+> ## Zipf's Law
+>
+> is an [empirical law](https://en.wikipedia.org/wiki/Empirical_law) formulated using [mathematical statistics](https://en.wikipedia.org/wiki/Mathematical_statistics) that refers to the fact that many types of data studied in the physical and social sciences can be approximated with a Zipfian distribution, one of a family of related discrete [power law](https://en.wikipedia.org/wiki/Power_law) [probability distributions](https://en.wikipedia.org/wiki/Probability_distribution).
```
Comment (tkphd) [inline _episodes/11-snakemake-intro.md]: Repeat "Zipf's Law" to make the first clause a sentence.

**Review by tkphd — CHANGES_REQUESTED** [review]
Looks good overall. Some minor cleanup requested.

File: _episodes/11-snakemake-intro.md (hunk 1 above)
Comment (psteinb) [reply _episodes/11-snakemake-intro.md]: turns out it's quite hard to do the 80/90 column rule when the markdown consists of links (shown characters versus not-shown characters)

File: _episodes/11-snakemake-intro.md (hunk 2 above)
Comment (psteinb) [reply _episodes/11-snakemake-intro.md]: fixed with 9611453

File: _episodes/11-snakemake-intro.md (hunk 3 above)
Comment (psteinb) [reply _episodes/11-snakemake-intro.md]: see 7e8b0b8

File: _episodes/11-snakemake-intro.md (hunk 1 above)
Comment (psteinb) [reply _episodes/11-snakemake-intro.md]: can you please check if that is ok?

**Review by tkphd — APPROVED** [review]
LGTM. Thanks for the contribution, @psteinb!

**Pull request merged**